modules/importing-a-custom-workbench-image.adoc

```diff
@@ -0,0 +1,75 @@
+:_module-type: PROCEDURE
+
+[id='importing-a-custom-workbench-image_{context}']
+= Importing a custom workbench image
+
+[role='_abstract']
+ifdef::upstream[]
+You can import custom workbench images that cater to your {productname-short} project's specific requirements. From the *Notebook images* page, you can enable or disable a previously imported workbench image and create an accelerator profile as a recommended accelerator for existing notebook images.
+endif::[]
+ifndef::upstream[]
+In addition to workbench images provided and supported by {org-name} and independent software vendors (ISVs), you can import custom workbench images that cater to your project's specific requirements.
+endif::[]
+
+After you have built a custom image and placed it in an image registry that is accessible to {productname-long}, you must import it so that your {productname-short} users (data scientists) can access it when they create a project workbench.
```

Comment:
Could this just be "you must import it so that your data scientist users can access it when they create a project workbench." ?